feature/experimental/system/health/tests/system_generic_health_check/metadata.textproto

```diff
@@ -0,0 +1,19 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid:  "a01b6e65-3b45-4e0c-a71d-f451c07fec6b"
+plan_id:  "Health-1.1"
+description:  "Generic Health Check"
+testbed:  TESTBED_DUT
+platform_exceptions: {
+  platform: {
+    vendor: JUNIPER
+    hardware_model: "PTX10008"
```

Comment:
`hardware_model` field is now deprecated and replaced by hardware_model_regex.  

If these deviations apply to all hardware models for Juniper then you need not specify any hardware_model. 

platform_exceptions will look like:

```
platform_exceptions: {
  platform: {
    vendor: JUNIPER
  }
  deviations: {
    linecard_cpu_utilization_unsupported: true
    consistent_component_names_unsupported: true
  }
}
```

Reply:
Thanks for your comment. Updated as:
```
platform_exceptions: {
  platform: {
    vendor: JUNIPER
  }
  deviations: {
    controller_card_cpu_utilization_unsupported: true
    linecard_cpu_utilization_unsupported: true
    consistent_component_names_unsupported: true
    fabric_drop_counter_unsupported: true
  }
}
```